readme.md

```diff
@@ -135,6 +155,7 @@ The debug will show you an array like this:
 
 ### Differences
 
+* No authentication layer is provided, you will need to build this yourself.
```

Comment:
I don't see any harm in driving home one of the key aspects of the package. Plus if someone scans the readme, or is just looking at one section, repeating yourself could be valuable.